i18n/fr.jsonp.js

```diff
@@ -19,8 +19,14 @@ document.localeJson = {
       "configure-title": "Configuration",
       "configure-about-usage-link": "Informations (Utilisation)",
       "configure-btn-library": "Bibliothèque ZIM",
+      "configure-btn-folderselect": "Sélectionner un dossier",
+      "configure-btn-rescan": "Rechercher",
```

Comment:
Change to "Réanalyser" like line below. Using "Rechercher" is ambiguous, because it means "search", and is the term used elsewhere in the app for article/title search.